<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QAnalystClass</class>
 <widget class="QWidget" name="QAnalystClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>967</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QAnalyst</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="5">
      <widget class="QPushButton" name="tickStatisButton">
       <property name="text">
        <string>统计tick</string>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QPushButton" name="delBtn">
       <property name="text">
        <string>删除</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QComboBox" name="columnListCtrl"/>
     </item>
     <item row="0" column="8">
      <widget class="QPushButton" name="btnNextBidJump">
       <property name="text">
        <string>NextBidJump</string>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="QLineEdit" name="timeGotoCtrl">
       <property name="minimumSize">
        <size>
         <width>160</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QPushButton" name="saveBtn">
       <property name="text">
        <string>保存</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QDateTimeEdit" name="beginTimeCtrl">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="dateTime">
        <datetime>
         <hour>0</hour>
         <minute>0</minute>
         <second>0</second>
         <year>2023</year>
         <month>1</month>
         <day>1</day>
        </datetime>
       </property>
       <property name="displayFormat">
        <string>yyyy/MM/dd HH:mm</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="10">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="7">
      <widget class="QPushButton" name="timeGotoButton">
       <property name="text">
        <string>GoTo</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>to:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QDateTimeEdit" name="endTimeCtrl">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="displayFormat">
        <string>yyyy/MM/dd HH:mm</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QComboBox" name="codeIdCtrl"/>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="goBtn">
       <property name="text">
        <string>查询</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QComboBox" name="klineTypeCtrl"/>
     </item>
     <item row="0" column="2">
      <widget class="QComboBox" name="lineTypeCtrl"/>
     </item>
     <item row="1" column="6">
      <widget class="QLabel" name="resultlabel">
       <property name="text">
        <string>-----------</string>
       </property>
      </widget>
     </item>
     <item row="0" column="9">
      <widget class="QPushButton" name="btnNextAskJump">
       <property name="text">
        <string>NextAskJump</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QTableView" name="tableViewCtrl"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QAnalyst.qrc"/>
 </resources>
 <connections/>
</ui>
